src/engine/cuecontrol.cpp

```diff
@@ -117,34 +117,64 @@ CueControl::CueControl(QString group,
     m_pCueIndicator = new ControlIndicator(ConfigKey(group, "cue_indicator"));
     m_pPlayIndicator = new ControlIndicator(ConfigKey(group, "play_indicator"));
 
-    m_pIntroPosition = new ControlObject(ConfigKey(group, "intro_position"));
-    m_pIntroPosition->set(-1.0);
+    m_pIntroStartPosition = new ControlObject(ConfigKey(group, "intro_start_position"));
+    m_pIntroStartPosition->set(-1.0);
 
-    m_pIntroSource = new ControlObject(ConfigKey(group, "intro_source"));
+    m_pIntroStartSource = new ControlObject(ConfigKey(group, "intro_start_source"));
 
-    m_pIntroSet = new ControlPushButton(ConfigKey(group, "intro_set"));
-    connect(m_pIntroSet, SIGNAL(valueChanged(double)),
-            this, SLOT(introSet(double)),
+    m_pIntroStartSet = new ControlPushButton(ConfigKey(group, "intro_start_set"));
+    connect(m_pIntroStartSet, SIGNAL(valueChanged(double)),
+            this, SLOT(introStartSet(double)),
             Qt::DirectConnection);
 
-    m_pIntroClear = new ControlPushButton(ConfigKey(group, "intro_clear"));
-    connect(m_pIntroClear, SIGNAL(valueChanged(double)),
-            this, SLOT(introClear(double)),
+    m_pIntroStartClear = new ControlPushButton(ConfigKey(group, "intro_start_clear"));
+    connect(m_pIntroStartClear, SIGNAL(valueChanged(double)),
```

Comment:
Please use Qt 5 function pointer syntax for new signal-slot connections

Reply:
OK Will do